lib/policyEvaluator/utils/conditions.js

```diff
@@ -232,12 +252,21 @@ conditions.convertConditionOperator = operator => {
                 // eslint-disable-next-line new-cap
                 return !operatorMap.StringEqualsIgnoreCase(key, value);
             },
-        StringLike: function stringLike(key, value) {
-            return value.some(item => {
-                const wildItem = handleWildcards(item);
-                const wildRegEx = new RegExp(wildItem);
-                return wildRegEx.test(key);
-            });
+        StringLike: function stringLike(key, value, prefix) {
+            function policyValRegex(testKey) {
+                return value.some(item => {
+                    const wildItem = handleWildcards(item);
+                    const wildRegEx = new RegExp(wildItem);
+                    return wildRegEx.test(testKey);
+                });
+            }
+            if (prefix === 'ForAnyValue') {
+                return key.some(k => policyValRegex(k));
```

Comment:
should just be able to pass in the method here directly.
```suggestion
                return key.some(policyValRegex);
```